cmd/e2e_test.go

```diff
@@ -9,12 +9,9 @@ import (
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
-	_ "github.com/redhat-appstudio/e2e-tests/tests/build"
-	_ "github.com/redhat-appstudio/e2e-tests/tests/cluster-registration"
-	_ "github.com/redhat-appstudio/e2e-tests/tests/e2e-demos"
+
+	// TODO: Once all controllers are working on KCP activate all the current tests.
 	_ "github.com/redhat-appstudio/e2e-tests/tests/has"
```

Comment:
Shouldn't we, instead of this, use ginkgo labels to select which tests are running? 
I didn't dig deep enough to see if it would be trivial change, or some bigger change would be needed in magefiles. If it would be bigger change, let's not do it in this PR.

Reply:
No, we cannot do that because the framework will initalize all the controllers and not all are ready in KCP and this will make the framework to die.